Tools/autotest/arducopter.py

```diff
@@ -6939,11 +6939,6 @@ def tests2b(self):  # this block currently around 9.5mins here
                  "Ensure position doesn't zero when GPS lost",
                  self.test_copter_gps_zero),
 
-            Test("DynamicRpmNotches",
```

Comment:
Why have you removed this?

Reply:
This test obviously will not pass when HAL_WITH_ESC_TELEM == 0. this last commit should not get merged, it is just there to get CI to run.